### ISSUE TITLE
Regarding the issue where the GPU freezes and becomes unresponsive right after the training starts

### ISSUE THREAD

**Issue by hj950815** [MIC-DKFZ/nnUNet#2287]
after running for two epochs, it seems like the program has frozen. I used the commands " watch -n1 nvidia-smi" and " watch -n1 gpustat", but neither of these commands refresh the status every second anymore. When I open a new teriminal and type nvidia-smi, I can no longer see the GPU status.

2024-06-13 12:14:23.083728: unpacking dataset...
2024-06-13 12:14:26.715768: unpacking done...
2024-06-13 12:14:26.716548: Unable to plot network architecture: nnUNet_compile is enabled!
2024-06-13 12:14:26.720917: 
2024-06-13 12:14:26.720984: Epoch 0
2024-06-13 12:14:26.721097: Current learning rate: 0.01
2024-06-13 12:15:47.507211: train_loss -0.3658
2024-06-13 12:15:47.507318: val_loss -0.66
2024-06-13 12:15:47.507349: Pseudo dice [0.8258, 0.7715, 0.784]
2024-06-13 12:15:47.507376: Epoch time: 80.79 s
2024-06-13 12:15:47.507394: Yayy! New best EMA pseudo Dice: 0.7938
2024-06-13 12:15:48.199892: 
2024-06-13 12:15:48.199970: Epoch 1
2024-06-13 12:15:48.200015: Current learning rate: 0.00999
2024-06-13 12:16:33.403986: train_loss -0.6333
2024-06-13 12:16:33.404450: val_loss -0.75
2024-06-13 12:16:33.404571: Pseudo dice [0.8895, 0.7969, 0.8242]
2024-06-13 12:16:33.404674: Epoch time: 45.2 s
2024-06-13 12:16:33.404744: Yayy! New best EMA pseudo Dice: 0.7981
2024-06-13 12:16:34.356774: 
2024-06-13 12:16:34.356873: Epoch 2
2024-06-13 12:16:34.356936: Current learning rate: 0.00998
^C^C^C

And I can not kill train process and "watch -n1 nvidia-smi"  with Crtl + C

Use top can find gnome-shell  always exists and I cannot kill it

   PID USER      PR  NI    VIRT    RES    SHR S  %CPU  %MEM     TIME+ COMMAND                                                          
   4210 hj        20   0 4227972 343700 139504 R  99.7   0.5  12:40.47 gnome-shell                                                      
   4166 hj        20   0   34744  27612   8640 S   0.7   0.0   0:00.22 gunicorn                                                         
   1815 root      20   0 1239832  35136  16128 S   0.3   0.1   0:08.94 ToDesk_Service                                                   
   4247 hj        20   0   94548  76476  18048 S   0.3   0.1   0:00.92 gunicorn                                                         
   4256 hj        20   0   94512  75796  17664 S   0.3   0.1   0:00.94 gunicorn                                                         
   4477 hj        20   0 3235936 272696 104832 S   0.3   0.4   0:07.58 sunloginclient                                                   
  11986 hj        20   0   16732   4608   3456 R   0.3   0.0   0:01.64 top                                                              
      1 root      20   0  168536  12240   7824 S   0.0   0.0   0:00.92 systemd           

Can you give me some advice, Thanks
